rosdep/python.yaml

```diff
@@ -3811,6 +3818,9 @@ python-sqlalchemy:
     wily: [python-sqlalchemy]
     xenial: [python-sqlalchemy]
     xenial_python3: [python3-sqlalchemy]
+python-statsd:
+  debian: [python-statsd]
```

Comment:
```suggestion
  debian: [python-statsd]
  fedora: [python-statsd]
```

Reply:
I wish the suggestions UI had a better way to suggest additional lines, but this works...